### PR TITLE
hints for grass planting & milking order

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: solutions/bronze/usaco-832.mdx [hunk 1]
```diff
@@ -5,11 +5,18 @@ title: Milking Order
 author: Ananth Kashyap
 ---
 
-## Solution 1
+<Spoiler title="Hint 1">
+Let's try placing cow $1$ at a specific position, how might we go about checking to see if we'll end up with a valid ordering?
+</Spoiler>
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:10]:
```suggestion
<Spoiler title="Hint 1">

Let's try placing cow $1$ at a specific position.
How might we go about checking to see if we'll end up with a valid ordering?

</Spoiler>

File: solutions/bronze/usaco-894.mdx [hunk 2]
```diff
@@ -5,10 +5,11 @@ title: Grass Planting
 author: Neo Wang, Aadit Ambadkar, Sandeep Wadhwa, Kevin Sheng, Nikhil Chatterjee
 ---
 
-[Official Analysis (C++)](http://www.usaco.org/current/data/sol_planting_silver_jan19.html)
-
-## Video Solution
+<Spoiler title="Hint 1">
+How many types of grass will a node with $x$ neighbors need?
+</Spoiler>
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-894.mdx:10]:
```suggestion
<Spoiler title="Hint 1">

How many types of grass will a node with $x$ neighbors need?

</Spoiler>
```

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/bronze/usaco-894.mdx [hunk 3]
```diff
@@ -5,10 +5,11 @@ title: Grass Planting
...
 
+<Spoiler title="Video Solution">
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-894.mdx:12]: wait idt we need spoilers for the video sol since they have to click play anyways

Comment (ryanchou-dev) [reply solutions/bronze/usaco-894.mdx:14]: i think for some videos the thumbnail is some diagram which might give stuff away? we have to keep the styling consistent right?

Comment (SansPapyrus683) [reply solutions/bronze/usaco-894.mdx:14]: yeah and in #3430 (the template) uh i didn't put a spoiler for the vid sol

Comment (ryanchou-dev) [reply solutions/bronze/usaco-894.mdx:14]: ah okayy

**Review by envyaims — APPROVED** [review]
ok

**Pull request merged**